various points from test users

buddy-web3 opened the pull request.

quinn-dougherty reviewed (approved):
LGTM! 

When I [jotted down those notes](https://github.com/quinn-dougherty/rallypoint/discussions/58) i didn't think of them as well-scoped issues, I thought of them as things we'd need to discuss. So it's super cool you were able to just interpret them as well-scoped issues already! it might be good to make some indication of what's been addressed and what hasn't?

buddy-web3 commented:
@quinn-dougherty here's a list:

- A fresh PostPage.tsx is too low on the screen.
- Added creation date to PostCards on home page
- Added short description to PostCards on home page
- make whole card clickable, not just title (for claimslist and postslist)
- create should redirect to logoin/singup if not authenticated
- Claim should redirect to logoin/singup if not authenticated
